_posts/0000-01-02-Memphis1983.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![](https://nerdymemes.com/wp-content/uploads/2018/10/Github%20is%20down.jpg)
*Programming*

Reply:
This looks good 👍